test/cypress/components/filterable-list-control.js

```diff
@@ -0,0 +1,83 @@
+export class FilterableListControl {
+
+  filterableListControl() {
+    return cy.get( '.o-filterable-list-controls' );
+  }
+
+  filterableElement( selector ) {
+    return this.filterableListControl().get( selector );
+  }
+
+  filterableListElement( name ) {
+    return cy.get( `#o-filterable-list-controls_${ name }` );
+  }
+
+  filterItemName( name ) {
+    return this.filterableListElement( 'title' ).type( name );
+  }
+
+  filterFromDate( date ) {
+    return this.filterableListElement( 'from-date' ).type( date );
+  }
+
+  filterToDate( date ) {
+    return this.filterableListElement( 'to-date' ).type( date );
+  }
+
+  resultsContent() {
+    return cy.get( '.o-post-preview_content' );
+  }
+
+  firstResultContent() {
+    return this.resultsContent().first();
+  }
+
+  lastResultContent() {
+    return this.resultsContent().last();
+  }
+
+  resultsHeader( name ) {
+    return cy.get( `.o-post-preview .m-meta-header_${ name }` );
+  }
+
+  resultsHeaderLeft() {
+    return this.resultsHeader( 'left' );
+  }
+
+  resultsHeaderRight() {
+    return this.resultsHeader( 'right' );
+  }
+
+  firstResultHeader() {
+    return this.resultsHeaderRight().first();
+  }
+
+  lastResultHeader() {
+    return this.resultsHeaderRight().last();
+  }
+
+  notification() {
+    return cy.get( '.o-filterable-list_notification' );
+  }
+
+  open() {
+    return this.showFilters();
+  }
+
+  applyFilters() {
+    return cy.get( 'form[action="."]' ).submit();
+  }
+
+  clearFilters() {
+    cy.get( '.a-btn__warning' ).click( { force: true } );
+  }
+
+  showFilters() {
+    return cy.get( '.o-expandable_cue-open' ).click( { force: true } );
```

Comment:
This is now failing on the browse filterable page tests because, to Ans's longstanding chagrin, the mobile section navigation menu is a version of an expandable, so `cy.get( '.o-expandable_cue-open' )` is returning two objects, resulting in the following error when trying to `click`:

> CypressError: `cy.click()` can only be called on a single element. Your subject contained 2 elements. Pass `{ multiple: true }` if you want to serially click each element.

You could fix it by adding `.o-filterable-list` to the front of the selector:

```suggestion
    return cy.get( '.o-filterable-list .o-expandable_cue-open' ).click();
```

(And I still think we don't want to force it.)

Reply:
OK, this is not failing now for some reason… :shrug: I think my suggestion is still safer, though.

Reply:
I fixed CypressError: cy.click() can only be called on a single element